cypress/support/utils.js

```diff
@@ -9,22 +12,16 @@ const WAIT_FOR_FRAME_TO_RENDER = 1000;
 
 const getCodeEditor = () => cy.get('.CodeMirror-code');
 
-export const getPreviewFrames = () => cy.get('[data-testid="previewFrame"]');
-
-export const getPreviewFrameNames = () => cy.get('[data-testid="frameName"]');
+const getPreviewFrameNames = () => cy.get('[data-testid="frameName"]');
 
-export const getFirstFrame = () => getPreviewFrames().first();
+export const getFirstFrame = () =>
+  cy.frameLoaded('[data-testid="previewFrame"]:first');
```

Comment:
Just merged [some changes to these utils](https://github.com/seek-oss/playroom/pull/328) today that simplify some of the iframe handling, so we might not need `cypress-iframe` anymore.

Reply:
In fact given the state of that package I'd rather not depend on it if we don't have to.

Reply:
I still need what `cypress-iframe` has to offer, but since the package is so stale I'll write it myself.